templates/note.php

```diff
@@ -1,3 +1,4 @@
 <div id="app-navigation-toggle" class="icon-menu" style="display:none;"></div>
 	<pre editor notes-timeout-change="save()" notes-autofocus ng-class="{saving: isSaving()}">{{note.content}}</pre>
+	<div class="note-meta">{{note.content | wordCount }} words</div>
```

Comment:
Forgot: This needs to be translatable. Ideally with the singular form included when there is 1 word. And we do not need to show anything when there are 0 words.

@DeepDiver1975 how can the singular/plural forms be distinguished?